### PR TITLE
Translate"Trusted Committer"into Chinese

### PR DESCRIPTION
**Pull request by yvonneT1**

### CODE REVIEW

**Comment by rrrutledge** [comment]
Hi, @yvonneT1, I've tried to reach out in the InnerSource community for a reviewer for this content, but haven't run across anyone, yet.  Do you have a professional contact at another company that could review this work?

Thank you.

**Comment by tanzhongyibidu** [comment]
russ，I will take a look also

**Comment by rrrutledge** [comment]
Oh, thank you so much, @tanzhongyibidu!

File: trusted-committer/CN/01-introduction-zh.asciidoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+受信任提交者角色是InnerSource社区中的关键角色之一。将受信任提交者视为您在社区里所相信的人，他们可以通过重要的技术决策和指导贡献者来获得最终收益。受信任提交者受到的要求很高，但是它是有高回报的。它并不是一个自以为是的把关者，对任何InnerSource社区的成功都至关重要。
+
+一般而言，受信任提交者角色是由其职责而不是特权定义的。在很高程度上，可信任提交者代表其InnerSource社区和该社区正在开发的产品的利益。他们负责社区和产品的健康。因此，作为可信任提交者，您将同时承担面向技术和社区的责任。我们将在以下各节中探讨这两个方面。
+
+在详细介绍受信任提交者（Trusted Committer）实际功能之前，让我们先从从高度抽象的角度将Trusted Committer角色与InnerSource中的其他角色进行对比，并解释为什么我们认为这个角色很重要。让我们从 https://innersourcecommons.org/resources/learningpath/contributor/index[_贡献者 Contributor_]角色开始。顾名思义，贡献者为InnerSource社区做出了贡献。这些贡献可能是代码或非代码工件，例如错误报告，功能请求或文档。
+
+贡献者可能是社区的一部分，也可能不是。它们可能是由另一个团队发送来开发团队所需的功能的。这就是为什么我们有时也将贡献者称为“来宾”或“来宾团队”的一部分。贡献者负责“适应”并达到社区的期望。
+
+受信任提交者始终是InnerSource社区的成员，该社区有时也称为宿主团队。我们这样比喻一下，受信任提交者既负责建造房屋，也负责制定房屋规则，以确保其客人舒适并可以进行有效的交流、合作。与贡献者相比，受信任提交者已经承担了将代码推向生产的责任，并且通常被允许执行更高风险性的任务。
```
Comment (adadai88) [inline trusted-committer/CN/01-introduction-zh.asciidoc:13]: 这里宿主团队： should be 维护团队，请修订。 

我们将host team and guest team翻译成专有名词：调用团队(guest team) ，称团队B为 维护团队(host team)

File: trusted-committer/CN/01-introduction-zh.asciidoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### 受信任提交者（Trusted Committer）
+
+受信任提交者角色是InnerSource社区中的关键角色之一。将受信任提交者视为您在社区里所相信的人，他们可以通过重要的技术决策和指导贡献者来获得最终收益。受信任提交者受到的要求很高，但是它是有高回报的。它并不是一个自以为是的把关者，对任何InnerSource社区的成功都至关重要。
+
+一般而言，受信任提交者角色是由其职责而不是特权定义的。在很高程度上，可信任提交者代表其InnerSource社区和该社区正在开发的产品的利益。他们负责社区和产品的健康。因此，作为可信任提交者，您将同时承担面向技术和社区的责任。我们将在以下各节中探讨这两个方面。
+
+在详细介绍受信任提交者（Trusted Committer）实际功能之前，让我们先从从高度抽象的角度将Trusted Committer角色与InnerSource中的其他角色进行对比，并解释为什么我们认为这个角色很重要。让我们从 https://innersourcecommons.org/resources/learningpath/contributor/index[_贡献者 Contributor_]角色开始。顾名思义，贡献者为InnerSource社区做出了贡献。这些贡献可能是代码或非代码工件，例如错误报告，功能请求或文档。
+
+贡献者可能是社区的一部分，也可能不是。它们可能是由另一个团队发送来开发团队所需的功能的。这就是为什么我们有时也将贡献者称为“来宾”或“来宾团队”的一部分。贡献者负责“适应”并达到社区的期望。
```
Comment (adadai88) [inline trusted-committer/CN/01-introduction-zh.asciidoc:11]: “来宾”或“来宾团队”： 请修订成  “调用”或“调用团队”

我们将host team and guest team翻译成专有名词：调用团队(guest team) ，称团队B为 维护团队(host team)

File: trusted-committer/CN/03-keeping-the-community-healthy-zh.asciidoc [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+人们留下来的另一个更关键的原因是，他们喜欢与社区中的其他成员（包括受信任的提交者）一起工作。在一个繁荣的社区里成员间通常会相互尊重和相互沟通。贡献被视为礼物或捐赠，而不是娱乐，而出色的（尤其是第一）贡献通常会受到称赞。tc在所有这方面的工作主要是为他人树立榜样，就是为预期的软件质量水平树立榜样。如有必要，受信任的提交者应为社区制定并制定行为准则。如果有某些社区成员的行为对社区健康有害或有害，那么受信任提交者有责任解决此​​问题。受信任提交者应为人们创造机会，使他们能够定期（面对面或虚拟地）聚在一起，彼此了解并在冲突发生时和平解决。
+人们还倾向于留下来，因为在InnerSource社区工作是获得新技能和个人成长的绝好机会。这再次是受信任提交者的角色真正重要的地方。受信任提交者通常成为初级开发人员的指导者，并且在拉动请求期间明确地花费时间去指出了需要改进的地方，还详细说明了为什么需要改进以及如何进行改进。他们提供了更改背后的理论或经验，并为实施更改的最佳方法提供了建议。这样，受信任提交者可以提高成员社区中的学习速度，远远超过传统软件开发项目中的学习速度。
+我们认为，受信任提交者应在 https://innersourcecommons.org/resources/learningpath/contributor/index[_贡献者 Contributor_]请求期间优先考虑批准和指导，而不要等到沟通的发布日期，除非有特别充分的理由不这样做。拉取请求期间的良好指导可提高贡献者的信任度和参与度，因此吸引更多的贡献。我们将在 https://innersourcecommons.org/resources/learningpath/trusted-committer/04/[_ Uplevelling Community Members 提升社区成员_]中对此进行更多讨论。
+最后，有些人会停留在InnerSource社区中，是因为他们开始专注于开发软件，而不是去做一些不实际、浪费的工作，尤其是在大型公司中，他们太注重流程。在此背景下，受信任提交者的工作是通过交流和制定有用的贡献准则来确保贡献者真正专注于其项目。
+为了实现上面所说的效果，应好好解释在拉取请求中我们所说的信号传递：注释应该是什么样的？我喜欢或+1评论是什么意思？ @提及带有/ CC前缀的人与使用/ FYI前缀有何不同？通常来说，tc需要确保在贡献者的贡献过程中不会出现太多问题，应支持社区识别和解决问题。最终，受信任的提交者应授权其社区发现与过程相关的问题，并尽可能地适应和改进它们。
```
Comment (adadai88) [inline trusted-committer/CN/03-keeping-the-community-healthy-zh.asciidoc:10]: tc: 此处不应该简写，should keep the full name as  受信任提交者

**Comment by rrrutledge** [comment]
@adadai88 do you have any comment here?  Thank you!

File: trusted-committer/CN/01-introduction-zh.asciidoc [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# 介绍
+
+### 受信任提交者（Trusted Committer）
+
+受信任提交者角色是InnerSource社区中的关键角色之一。将受信任提交者视为您在社区里所相信的人，他们可以通过重要的技术决策和指导贡献者来获得最终收益。受信任提交者受到的要求很高，但是它是有高回报的。它并不是一个自以为是的把关者，对任何InnerSource社区的成功都至关重要。
```
Comment (tanzhongyi003) [inline trusted-committer/CN/01-introduction-zh.asciidoc:5]: 他们可以通过重要的技术决策和对贡献者进行指导来获得最终收益

Comment (tanzhongyi003) [reply trusted-committer/CN/01-introduction-zh.asciidoc:5]: 对受信任提交者的要求很高，

File: trusted-committer/CN/01-introduction-zh.asciidoc [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+一般而言，受信任提交者角色是由其职责而不是特权定义的。在很高程度上，可信任提交者代表其InnerSource社区和该社区正在开发的产品的利益。他们负责社区和产品的健康。因此，作为可信任提交者，您将同时承担面向技术和社区的责任。我们将在以下各节中探讨这两个方面。
+
+在详细介绍受信任提交者（Trusted Committer）实际功能之前，让我们先从从高度抽象的角度将Trusted Committer角色与InnerSource中的其他角色进行对比，并解释为什么我们认为这个角色很重要。让我们从 https://innersourcecommons.org/resources/learningpath/contributor/index[_贡献者 Contributor_]角色开始。顾名思义，贡献者为InnerSource社区做出了贡献。这些贡献可能是代码或非代码工件，例如错误报告，功能请求或文档。
```
Comment (tanzhongyi003) [inline trusted-committer/CN/01-introduction-zh.asciidoc:9]: 受信任提交者（Trusted Committer）建议都用这种中文+英文的方式

File: trusted-committer/CN/01-introduction-zh.asciidoc [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+贡献者可能是社区的一部分，也可能不是。它们可能是由另一个团队发送来开发团队所需的功能的。这就是为什么我们有时也将贡献者称为“调用者”或“调用团队”的一部分。贡献者负责“适应”并达到社区的期望。
```
Comment (tanzhongyi003) [inline trusted-committer/CN/01-introduction-zh.asciidoc:11]: 它们可能是由另一个团队发送来开发团队所需的功能的。 感觉有些不通顺。

Comment (tanzhongyi003) [reply trusted-committer/CN/01-introduction-zh.asciidoc:11]: 贡献者负责“适应”并达到社区的期望， 建议可以改为贡献者（contributor）遵循并达到社区的要求。

**Comment by rrrutledge** [comment]
Thanks for the review, @tanzhongyi003 !

**Comment by tanzhongyi003** [comment]
I have discussed with the contributor offline, they will provide a more updated version soon.

**Comment by rrrutledge** [comment]
Sounds great!  Thank you!

**Review by tanzhongyi003 — COMMENTED** [review]
LGTM， let us move on and keep improving.

**Review by tanzhongyi003 — APPROVED** [review]

**Comment by rrrutledge** [comment]
Awesome!  Looks like it's ready to merge

**Review by lenucksi — APPROVED** [review]
Thank you for your great work everyone - @yvonneT1 for creating the translation and @adadai88 as well as @tanzhongyi003 for the detailed reviews! :tada: Gladly merging this now!

**Pull request merged**